pkgs/by-name/to/tomlq/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tomlq";
+  version = "0.1.6";
+
+  src = fetchFromGitHub {
+    owner = "cryptaliagy";
+    repo = "tomlq";
+    rev = "refs/tags/${version}";
+    hash = "sha256-g8xjz8qCTiulTwcEbLTHYldw4PI+4ZfCOMJs+J6L1C4=";
+  };
+
+  cargoHash = "sha256-/cepTVJoBM1LYZkFpH9UCvE74cSszHDaeThsZksQ1P8=";
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
Go ahead and also change the `with maintainers;` part to `with lib.maintainers;` and i'll commit this

Reply:
There's web editor in `Files changed` overview. Doing squash merge is just a grace, and originally according to the guidelines we were not supposed to do this.

Reply:
Oh okay, thanks!